gtdynamics/dynamics/Chain.h

```diff
@@ -93,6 +96,41 @@ class Chain {
    */
   Pose3 poe(const Vector &q, boost::optional<Pose3 &> fTe = boost::none,
             gtsam::OptionalJacobian<-1, -1> J = boost::none);
+
+  /**
+   * This function implements the Dynamical dependency between the
+   * joint torques and the wrench applied on the body FOR A 3-LINK CHAIN (tau =
+   * J*F) The input wrench is the wrench applied on the body by the joint
+   * closest to the body. This equation is true in the case of massless links in
+   * the chain. Detailed explanation in readme (chain.md)
+   *
+   * @param wrench .................. Wrench applied on the body by the joint
+   * closest to it in the chain.
+   * @param angles .................. Angles of the joints in the chain.
+   * @param torques ................. Torques applied by the joints.
+   * @return ........................ Vector of difference.
+   */
+  gtsam::Vector3 DynamicalEquality3(
+      const gtsam::Vector6 &wrench, const gtsam::Vector3 &angles,
+      const gtsam::Vector3 &torques,
+      gtsam::OptionalJacobian<3, 6> J0 = boost::none,
```

Comment:
We typically use `H` for these Jacobians, and that might also disambiguate it from the chain jacobian. Even better than H1, H2, H3  would be `H_wrench`, `H_angles`, `H_torques`

Reply:
done